parquet/src/main/java/org/apache/iceberg/parquet/RemoveIds.java

```diff
@@ -77,6 +77,16 @@ public Type primitive(PrimitiveType primitive) {
         .named(primitive.getName());
   }
 
+  @Override
+  public Type variant(GroupType variant) {
+    Types.GroupBuilder<GroupType> builder =
+        Types.buildGroup(variant.getRepetition()).as(variant.getLogicalTypeAnnotation());
+    for (Type field : variant.getFields()) {
+      builder.addField(field);
```

Comment:
If a VARIANT has a typed_value with IDs, do we need to visit children and remove the IDs on the children too?